docs/hooks/useTransactor.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# useTransactor
+
+Use this hook to interact with the chain and get UI feedback on the transaction status. You can pass in anything that is a valid parameter to [Viem's `sendTransaction` function](https://viem.sh/docs/actions/wallet/sendTransaction#parameters).
+
+```ts
+const transactor = useTransactor();
+const writeTx = transactor({
+  to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
```

Comment:
What address is this? The BG DAO address? lol just curious

Reply:
Also: let's use double quotes, following our prettier rules.

Reply:
Good question. I can't see where I got it from tbh. Should I instead put `buidlguidl.eth` or its' address (`0x97843608a00e2bbc75ab0C1911387E002565DEDE`)?

Reply:
maybe put its address (so its kind of copy-pastable) and maybe while in below description we could mention that it sends 1 ETH to buidlguidl.eth instead of whole address their

Reply:
Great idea! I added it here: https://github.com/scaffold-eth/se-2-docs/commit/aee53a3ed5465b9785e3b9fbbf00d03e1f8f3832